web-client/src/global.scss

```diff
@@ -29,6 +29,8 @@
   // Docs: https://ionicframework.com/docs/theming/themes#application-colors
   --ion-background-color: var(--ion-color-primary-background);
   --ion-background-color-rgb: var(--ion-color-primary-background-rgb);
+  --ion-toolbar-background: var(--ion-color-primary-background);
+  --ion-toolbar-color: var(--ion-color-white);
```

Comment:
The toolbar sits on top of the app and has its own background color.
Removing it will result in a color mismatch with the content background and the items inside it (title, icons, etc) will have "primary" as base color

Reply:
Ah, so it looks like this is actually an `ios` mode thing. We should probably think about what we want to do here: do we want to support and maintain both `ios` and `md` platform themes concurrently, or do we just want to fix the app mode / theme to `md`, for now?

1. If we support both, we should probably document `ios`-specific overrides like this one, and we'll probably want to get the `ios` variations covered in Chromatic too?
2. If we support `md` only, we can drop the `ios`-specific overrides, for now?

Reply:
(CC @longtomjr for thoughts?)

Reply:
Cool, capturing the discussion with @longtomjr and @jongbonga, we should probably keep `ios`-specific overrides in an `ios`-specific block, like documented [here](https://ionicframework.com/docs/theming/platform-styles#overriding-mode-styles):

```css
.ios {
  --ion-toolbar-background: var(--ion-color-primary-background);
  --ion-toolbar-color: var(--ion-color-white);
}
```

Reply:
Cool, that looks good!